components/SchemaInfoTooltip.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactNode, useRef, useState } from "react";
+import { Tooltip } from "@prisma/lens";
+
+type SchemaInfoTooltipProps = {
+  children: ReactNode;
+};
+
+const SchemaInfoTooltip = ({ children }: SchemaInfoTooltipProps) => {
+  const [isHovering, setIsHovering] = useState(false);
+  const [delayTooltipPhaseout, setDelayTooltipPhaseout] = useState<ReturnType<
+    typeof window.setTimeout
+  > | null>(null);
+
+  const handleMouseOver = () => {
+    clearTimeout(delayTooltipPhaseout as unknown as number);
+    setIsHovering(true);
+  };
```

Comment:
❓ 
I'm sure there is a way of specifying the correct type for the timeout, but I could not find it.
The cast to `unknown` is not very nice.

Reply:
Isn't just a text like we have now good enough? 
<img width="765" alt="CleanShot 2023-02-15 at 08 10 28@2x" src="https://user-images.githubusercontent.com/19674362/218957248-431c1345-3356-4e28-b237-09c082a1b46c.png">

Reply:
You are right; I'll change it back to the original line.

I thought there might be more "info" to be added in the future, but YAGNI

Reply:
@albingroen I will apply the fixup commits once the PR is approved